packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -116,30 +118,30 @@ class CachedTargetIdGenerator {
    * target ID has ever been assigned.
    */
   next(query: Query): TargetId {
```

Comment:
Did you play around with this to see what it would take to make `next`, `purged` and `cachedId` take a Target as an argument? It looks like that might be cleaner.

Reply:
Done.